Backend/src/bdd/resources/features/contact.feature

```diff
@@ -0,0 +1,11 @@
+Feature: Testing contact controller
+
+  Scenario: Test user add contact details - valid username
+    Given User calls add contact endpoint for valid username
```

Comment:
always write feature files which are easy for end users to read. End user doesnt understand what is add contact endpoint. Need to reframe all such sentences in the feature file